dialog/base.go

```diff
@@ -159,17 +174,22 @@ func (d *dialog) SetDismissText(label string) {
 	widget.Refresh(d.win)
 }
 
+func (d *dialog) hideWithResp(resp bool) {
```

Comment:
We don't usually truncate function names - surely hideWithResponse is not too verbose?